tests/unit/index.js

```diff
@@ -57,9 +57,9 @@ describe('ycb unit tests', function () {
                 .concat(readFixtureFile('simple-3.json'));
             ycb = new libycb.Ycb(bundle);
             cmp([0,0,0], ycb._parseContext({}));
-            cmp([0,7,0], ycb._parseContext({region:'fr'}));
-            cmp([0,6,9], ycb._parseContext({flavor:'bt', region:'ir'}));
-            cmp([2,7,8], ycb._parseContext({lang:'fr_FR', region:'fr', flavor:'att'}));
+            cmp([0,4,0], ycb._parseContext({region:'fr'}));
```

Comment:
why did these change?

Reply:
Unused dimension values with used ancestor values were getting assigned distinct labels to even though they are functionally equivalent to their ancestor. This change maps them to the ancestor instead.